react/week2/to_do_list/src/index.js

```diff
@@ -0,0 +1,177 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+
+const taskList = [
+  {
+    id: 0,
+    description: "Get out of bed",
+    color: "brown",
+    deadline: "2020-06-08",
+    done: false,
+    visibility: true
+  },
+  {
+    id: 1,
+    description: "Brush teeth",
+    color: "red",
+    deadline: "2020-06-08",
+    done: false,
+    visibility: true
+  },
+  {
+    id: 2,
+    description: "Eat breakfast",
+    color: "orange",
+    deadline: "2020-06-08",
+    done: true,
+    visibility: true
+  }
+];
+
+class Header extends React.Component {
+  render() {
+    return (
+      <div>
+        <h1> Welcome to My ToDo List</h1>
+        <br ></br>
+      </div>
+    )
+  }
+}
+
+class Counter extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      ellapsedTime: this.props.initial,
+    };
+  }
+
```

Comment:
It's not clear from the documentation but you can simplify the definition of state in your components to this: 
```suggestion
    state = {
      ellapsedTime: this.props.initial,
    };
```

React takes care of all the constructor magic behind the scenes!